liquidctl/driver/aquacomputer.py

```diff
@@ -0,0 +1,262 @@
+"""liquidctl driver for Aquacomputer family of watercooling devices.
+
+Aquacomputer D5 Next watercooling pump
+--------------------------------------
+The pump sends a status HID report every second with no initialization
+being required.
+
+The status HID report exposes sensor values such as liquid temperature and
+two groups of fan sensors, for the pump and the optionally connected fan.
+These groups provide RPM speed, voltage, current and power readings. The
+pump additionally exposes +5V and +12V voltage rail readings.
+
+Driver
+------
+Linux has the aquacomputer_d5next driver available since v5.15. Subsequent
+releases have more functionality and support a wider range of devices. If
+present, it's used instead of reading the status reports directly.
+
+Copyright (C) 2022 - Aleksa Savic
+
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+# uses the psf/black style
+
+import logging
+
+from liquidctl.driver.usb import UsbHidDriver
+from liquidctl.error import NotSupportedByDriver
+
+_LOGGER = logging.getLogger(__name__)
+
+_AQC_TEMP_SENSOR_DISCONNECTED = 0x7FFF
+_AQC_FAN_VOLTAGE_OFFSET = 0x02
+_AQC_FAN_CURRENT_OFFSET = 0x04
+_AQC_FAN_POWER_OFFSET = 0x06
+_AQC_FAN_SPEED_OFFSET = 0x08
+
+_AQC_STATUS_READ_ENDPOINT = 0x01
+
+
+def get_unaligned_be16(msg, offset):
+    return msg[offset] << 8 | msg[offset + 1]
```

Comment:
While this obviously works, I think the Python way is to use `int.from_bytes`.  It's a lot more verbose, but for that we already have a `liquidctl.util.u16be_from` wrapper.

Reply:
Agreed, `u16be_from` is better.